CheckDrive.Web/CheckDrive.Web/Views/MechanicHandovers/Create.cshtml

```diff
@@ -6,10 +6,11 @@
     bool noDriversAvailable = ViewBag.Drivers == null || !((IEnumerable<SelectListItem>)ViewBag.Drivers).Any();
     bool noCarsAvailable = ViewBag.Cars == null || !((IEnumerable<SelectListItem>)ViewBag.Cars).Any();
 }
-<div class="row mt-custom">
-    <div class="col-md-4 offset-md-1 mt-5">
-        <h4>Ro'yxatga olish</h4>
-        <hr class="w-50" />
+<form id="mechanicHandoverForm" asp-action="Create" method="post">
+    <div class="row mt-custom" style="width:580px">
+        <div class="col-md-4 offset-md-1 mt-1" style="width:580px">
+        @* <h4>Ro'yxatga olish</h4> *@
+        @* <hr class="w-50" /> *@
```

Comment:
if you don't use it, it's better to remove this commented line?

Reply:
Yes, I will remove it